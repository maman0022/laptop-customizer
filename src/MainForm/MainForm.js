import React from 'react';
import Specs from '../Specs/Specs'
import './MainForm.css'

export default function MainForm(props) {
  return (
    <form className="main__form">
      <h2>Customize your laptop</h2>
      <Specs USCurrencyFormat={props.USCurrencyFormat} state={props.state} updateFeature={props.updateFeature} store={props.store}/>
    </form>
  )
}