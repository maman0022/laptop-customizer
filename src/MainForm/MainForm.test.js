import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MainForm from './MainForm'

describe('MainForm Component', () => {
  it('the form renders with a header and reference to Specs component', () => {
    const wrapper = shallow(<MainForm/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})