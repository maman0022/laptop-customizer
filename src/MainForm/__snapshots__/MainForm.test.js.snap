// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainForm Component the form renders with a header and reference to Specs component 1`] = `
<form
  className="main__form"
>
  <h2>
    Customize your laptop
  </h2>
  <Specs />
</form>
`;
